src/main/java/edu/bu/metcs673/trackr/controller/BankAccountController.java

```diff
@@ -87,4 +84,28 @@ public ResponseEntity<GenericApiResponse> modifyBankAccount(@PathVariable(value
                         MessageFormat.format(CommonConstants.DEACTIVATE_BANK_ACCOUNT, String.valueOf(id))),
                 HttpStatus.OK);
     }
+    @GetMapping("/find/{userId}/{bankAccountId}")
```

Comment:
Your API paths are not RESTful please take a look at this doc to fix them.
https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design